tests/storage/restore.go

```diff
@@ -301,14 +301,16 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 		})
 
 		Context("and no snapshot", func() {
-			It("[test_id:5255]should reject restore", func() {
```

Comment:
Do we need to keep the test is?

Reply:
Maybe add an entirely different test and remove this one in a separate commit

Reply:
OK